src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/info/BlobContainerInfoContributor.java

```diff
@@ -15,18 +15,18 @@
 @Profile(Profiles.NOT_STORAGE_STUB)
 public class BlobContainerInfoContributor implements InfoContributor {
 
-    private final CloudBlobClient client;
+    private final BlobServiceClient client;
 
-    public BlobContainerInfoContributor(CloudBlobClient client) {
+    public BlobContainerInfoContributor(BlobServiceClient client) {
         this.client = client;
     }
 
     @Override
     public void contribute(Builder builder) {
         builder.withDetail(
             "containers",
-            StreamSupport.stream(client.listContainers().spliterator(), false)
-                .map(CloudBlobContainer::getName)
+            StreamSupport.stream(client.listBlobContainers().spliterator(), false)
+                .map(BlobContainerItem::getName)
```

Comment:
looks like a perfect time to get rid of extra dependencies used here, right? we can just `client.listContainers().stream().map().collect()` ;)